src/build/app/mod.rs

```diff
@@ -1429,7 +1437,7 @@ impl<'b> App<'b> {
     }
 
     #[allow(clippy::debug_assert_with_mut_call)]
-    // used in clap_generate (https://github.com/clap-rs/clap_generate)
+    // PUB_HIDDEN: used in clap_generate
```

Comment:
This is little bit related to this PR but is not related to view.

Reply:
This pr is not about view only, see the rationale below.